docs/api_reference/v1beta1.en.md

```diff
@@ -1,6 +1,6 @@
 +++
 title = "v1beta1 API Reference"
-date = 2021-12-03T16:34:53+05:00
+date = 2021-12-08T15:09:59+05:00
```

Comment:
Don't need to change the date if the contents didn't changed (I know it's autogenerated, before committing it should be reset in that case).